### DISCUSSION TITLE
Aptos Testnet Indexer API

### DISCUSSION THREAD

**Discussion by MoonShiesty** [aptos-labs/aptos-developer-discussions#300] [Questions]
### Discord user ID

moonshiesty

### Describe your question in detail.

the aptos testnet indexer deployment returns an error that the query_root is not found around half the time:

```
{
  "errors": [
    {
      "message": "field 'current_collection_ownership_v2_view' not found in type: 'query_root'",
      "extensions": {
        "path": "$.selectionSet.current_collection_ownership_v2_view",
        "code": "validation-failed"
      }
    }
  ]
}
```
same error with the old query_root:
```
{
  "errors": [
    {
      "message": "field 'current_collections_v2' not found in type: 'query_root'",
      "extensions": {
        "path": "$.selectionSet.current_collections_v2",
        "code": "validation-failed"
      }
    }
  ]
}
```

seems like there's two different graphql deployments on testnet

### What have you tried or looked at?  Or how can we reproduce the error?

this query fails in the explorer around half the time:

```
query GetCollectionsByOwner($OWNER: String!) {
  current_collections_v2(where: { creator_address: { _eq: $OWNER } }) {
    collection_name
    collection_id
  }
}
```
```
{
  "errors": [
    {
      "message": "field 'current_collections_v2' not found in type: 'query_root'",
      "extensions": {
        "path": "$.selectionSet.current_collections_v2",
        "code": "validation-failed"
      }
    }
  ]
}
```
returns this around half the time:
```
{
  "data": {
    "current_collections_v2": []
  }
}
```
### Which SDK or tool are you using? (if any)

Indexer API

### Describe your environment or tooling in detail

aptos testnet hasura explorer

**Comment by banool** [comment]
Thanks for the report, we'll investigate soon!

**Reply by MoonShiesty** [reply]
thanks!

**Reply by banool** [reply]
This has hopefully been fixed. I believe this was due to a bad assumption (in regards to Hasura metadata) in our IaC that we have now fixed. Thanks again for letting us know!!

**Reply by MoonShiesty** [reply]
awesome, everything looks good!
